test/server_integration/http/ssl/config.js

```diff
@@ -9,13 +9,15 @@
 
 import { readFileSync } from 'fs';
 import { CA_CERT_PATH, KBN_CERT_PATH, KBN_KEY_PATH } from '@kbn/dev-utils';
+import { ScoutTestRunConfigCategory } from '@kbn/scout-info';
 import { createKibanaSupertestProvider } from '../../services';
 
 export default async function ({ readConfigFile }) {
   const httpConfig = await readConfigFile(require.resolve('../../config.base.js'));
   const certificateAuthorities = [readFileSync(CA_CERT_PATH)];
 
   return {
+    testConfigCategory: ScoutTestRunConfigCategory.UI_TEST,
```

Comment:
shouldn't it be `API_TEST`?

index file: 

```
  describe('kibana server with ssl', () => {
    it('handles requests using ssl', async () => {
      await supertest.get('/').expect(302);
    });
  });
```

Reply:
There's a follow-up PR for it already: #210783